lib/elixir/lib/exception.ex

```diff
@@ -1117,8 +1126,8 @@ defmodule MismatchedDelimiterError do
     |> Enum.intersperse("\n")
   end
 
-  defp mismatched_closing_delimiter(end_column),
-    do: [n_spaces(end_column - 1), red("└ mismatched closing delimiter")]
+  defp mismatched_closing_delimiter(end_column, expected_closing_delimiter),
+    do: [n_spaces(end_column - 1), red(~s/└ mismatched closing delimiter (expecting "#{expected_closing_delimiter}")/)]
```

Comment:
```suggestion
    do: [n_spaces(end_column - 1), red(~s/└ mismatched closing delimiter (expected "#{expected_closing_delimiter}")/)]
```

Expected reads better no? Since it is "mismatched"?